internals/cli/cmd_add_test.go

```diff
@@ -89,7 +89,7 @@ services:
 				args = append(args, "--combine")
 			}
 			args = append(args, "foo", path)
-			rest, err := cli.Parser(cli.Client()).ParseArgs(args)
+			rest, err := cli.ParserForTest().ParseArgs(args)
```

Comment:
@benhoyt I didn't find a cleaner way to pass `RunOptions` from `cli.Run` into `CmdOptions` (e.g. a global `RunOptions` to pass `RunOptions` from `cli.Run()` without changing the `cli.Parser()` parameter list sounds super ugly, as does introducing something like `cli.ParserWithRunOptions()`), so this diff grows quite large.

Therefore, I decided to centralize the creation of a "default parser" into `cli.ParserForTest()` so that in the future if/when `cli.Parser()`'s argument list changes again, the diff will be smaller (only `cli.ParserForTest()`'s body needs to be changed then).